frontend/integration-tests/tests/devconsole/topology.scenario.ts

```diff
@@ -0,0 +1,106 @@
+import { browser, $, Key, ExpectedConditions as until } from 'protractor';
+import { appHost, checkLogs, checkErrors } from '../../protractor.conf';
+
+import {
+  applicationSelector,
+  navigateImportFromGit,
+  setBuilderImage,
+  enterGitRepoUrl,
+  addApplication,
+  applicationName,
+  appName1,
+  appName,
+  builderImage,
+  buildImageVersion,
+  createButton,
+  builderImageVersionName,
+} from '../../views/devconsole-view/git-imort-flow';
+
+import {
+  topologyContainer,
+  topologyContent,
+  topologyGraph,
+  topologyToolbar,
+  baseNodes,
+  baseNodeLabels,
+  defaultGroupLabels,
+  deploymentConfigs,
+} from '../../views/devconsole-view/topology.view';
+
+import { newApplicationName, newAppName } from '../../views/devconsole-view/new-app-name.view';
+
+import {
+  switchPerspective,
+  Perspective,
+  sideHeader,
+} from '../../views/devconsole-view/dev-perspective.view';
+
+import { info } from 'console';
+
+
+describe('git import flow', () => {
+  let newApplication;
+  let newApp;
+  const importFromGitHeader = $('[data-test-id="resource-title"]');
+
+  beforeAll(async () => {
+    await browser.get(`${appHost}/k8s/cluster/projects`);
+    newApplication = newApplicationName();
+    newApp = newAppName();
+
+    await switchPerspective(Perspective.Developer);
+    expect(sideHeader.getText()).toContain('Developer');
+    await navigateImportFromGit();
+    await browser.wait(until.textToBePresentInElement(importFromGitHeader, 'Import from git'));
+    expect(importFromGitHeader.getText()).toContain('Import from git');
+    await enterGitRepoUrl('https://github.com/sclorg/nodejs-ex.git');
+
+    await appName.click();
+    expect(appName.getAttribute('value')).toContain('nodejs-ex-git');
+    await addApplication(newApplication, newApp);
+
+    applicationName.getAttribute('value').then(function (text) {
+      info("Verifying applicationName attribute value = [" , text, "] [", newApplication, "]");
+      expect(text).toContain(newApplication);
+    });
+
+    appName.getAttribute('value').then(function (text) {
+      info("Verifying appName attribute value = [" , text, "] [", newApp, "]");
+      expect(text).toContain(newApp);
+    });
+
+    await setBuilderImage(builderImageVersionName);
+    expect(builderImage.isSelected());
+    expect(buildImageVersion.getText()).toContain('8-RHOAR');
+    await browser.wait(until.elementToBeClickable(createButton), 5000);
+    expect(createButton.isEnabled());
+    await createButton.click();
+    await browser.wait(until.urlContains('topology/ns/default'));
+    expect(browser.getCurrentUrl()).toContain('topology/ns/default');
+  });
+
+  afterEach(() => {
+    checkLogs();
+    checkErrors();
+  });
+
+  it('topology normal flow', async () => {
```

Comment:
I've written some tests for an empty topology view. I think we can incorporate those changes.